go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/hashicorp/vault-plugin-auth-oci
+
+go 1.12
+
+require (
+	github.com/armon/go-radix v1.0.0 // indirect
+	github.com/frankban/quicktest v1.4.1 // indirect
+	github.com/go-test/deep v1.0.2 // indirect
+	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/hashicorp/errwrap v1.0.0
+	github.com/hashicorp/go-hclog v0.9.2
+	github.com/hashicorp/go-immutable-radix v1.1.0 // indirect
+	github.com/hashicorp/go-version v1.2.0 // indirect
+	github.com/hashicorp/golang-lru v0.5.3 // indirect
+	github.com/hashicorp/vault/api v1.0.4
+	github.com/hashicorp/vault/sdk v0.1.13
+	github.com/oracle/oci-go-sdk v7.0.0+incompatible
+	github.com/pierrec/lz4 v2.2.6+incompatible // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/stretchr/objx v0.2.0 // indirect
+	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4 // indirect
+	golang.org/x/net v0.0.0-20190813141303-74dc4d7220e7 // indirect
+	golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a // indirect
+	golang.org/x/text v0.3.2 // indirect
+	google.golang.org/genproto v0.0.0-20190801165951-fa694d86fc64 // indirect
+	google.golang.org/grpc v1.23.0 // indirect
```

Comment:
```suggestion
```
This will cause a conflict when build with Vault.

Reply:
Have removed it.